File: src/gestorAplicacion/gente/Cliente.java
package gestorAplicacion.gente;

import java.util.ArrayList; 
import java.util.Collections;
import java.io.Serializable;

public class Cliente extends Persona implements Serializable{
	private static ArrayList<Cliente> lista_socio = new ArrayList<Cliente>();
	
	public static ArrayList<Cliente> getListaSocios(){
		return lista_socio;
	}
	public static void addSocio(Cliente cliente) {
		lista_socio.add(cliente);
	}
	
	public String perfil() {
		return "El cliente es la razon de ser del restaurante, puesto que es el quien consume los productos que se venden en el";
	}
	
	//constructor
	public Cliente(int cedula,String nombre,int telefono) {
		super(cedula,nombre,telefono);
	}
	
	public String informacion() {
		return "nombre del cliente: "+this.getNombre()+" \ncedula: "+this.getCedula()+" \ntelefono: "+this.getTelefono();
	}
	
}

File: Python/gestorAplicacion/gente/cliente.py
from persona import Persona
class Cliente(Persona):
	_lista_socio=[]
	def __init__(self,cedula,nombre,telefono):
		super.__init__(self,cedula,nombre,telefono)
	
	#metodos get

	def getListaSocios(self):
	    return Cliente._lista_socio
	
	#otros metodos

    @classmethod    
    def addSocio(cls,cliente):
		Cliente._lista_socio.append(cliente)
	
	def informacion(self):
        return f"nombre del cliente: {self._nombre} \ncedula: {self._cedula} \ntelefono: {self._telefono}"